src/torcontrol.cpp

```diff
@@ -662,26 +662,26 @@ void TorController::reconnect_cb(evutil_socket_t fd, short what, void *arg)
 }
 
 /****** Thread ********/
-struct event_base *base;
```

Comment:
You are overreacting. The problem is elsewhere, of course. The original author is using `base` all over the source code and arguments of methods conflicts with the global (now static). I think that renaming the global and making it static is the best solution here. Yes, discovered because of turning on `-Wshadow`.